.github/workflows/ci_accountability.yml

```diff
@@ -8,6 +8,18 @@ on:
   pull_request:
     branches-ignore:
       - "chore/l10n*"
+    paths:
+      - "*"
```

Comment:
This may catch all the folders so I would add the list of individual files that you want to check.

Reply:
this should only catch the root files, but not the files inside folders. Again, the idea is to avoid problems when some of the files from the root folder is changed, as I'm not sure how to react depending on the modified file. As with the previous issue, this also can be improved in the future.

Reply:
Single `*` don't match `/` character, so they'd only apply to files in the root folder.